src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img class="card__item" src="images/imac.jpeg" alt="img">
+      </div>
+      <div class="card__body">
+        <div class="card__title">
+          <p>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+        </div>
+        <div class="card__code">
+          <p>Код товара: 195434</p>
+        </div>
+        <div class="card__sub-body">
+          <div class="card__stars">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
```

Comment:
make this a `stars` block and use modificator with number of stars, just like you did in `stars` task (`stars--1`, `stars--2`, ...)